packages/frontpage/lib/schema.ts

```diff
@@ -100,6 +129,25 @@ export const Comment = sqliteTable(
   }),
 );
 
+export const CommentAggregates = sqliteTable(
+  "comment_aggregates",
+  {
+    id: integer("id").primaryKey(),
+    commentId: integer("comment_id")
+      .notNull()
+      .references(() => Comment.id)
+      .unique(),
+    voteCount: integer("vote_count"),
+    rank: integer("rank"),
+    createdAt: dateIsoText("created_at")
+      .notNull()
+      .default(sql`(current_timestamp)`),
```

Comment:
```suggestion
      .default(sql`(strftime('%Y-%m-%dT%H:%M:%fZ', 'now'))`),
```

Applies to other columns too, then we can remove `createdAt: new Date()` in the insert queries.

Reply:
Have only updated defaults for the columns I've added. Will probably need a quick PR/migration to fix them up for the rest of the db at some point after.